src/lib/ngsi9/UpdateContextAvailabilitySubscriptionRequest.cpp

```diff
@@ -49,11 +49,11 @@ UpdateContextAvailabilitySubscriptionRequest::UpdateContextAvailabilitySubscript
 *
 * UpdateContextAvailabilitySubscriptionRequest::render - 
 */
-std::string UpdateContextAvailabilitySubscriptionRequest::render(RequestType requestType, const std::string& indent)
-{
+std::string UpdateContextAvailabilitySubscriptionRequest::render(const std::string& indent)
+{  
```

Comment:
As discussed at skype, let's consider style-check aspect in the last PR in the "refactor cycle". NTC